src/accounts_refactor.rs

```diff
@@ -281,31 +255,20 @@ impl<L: Ledger, Key: KeyPair + DeserializeOwned + Serialize> Accounts<L, Key> {
         Ok(self.store.revert_version()?)
     }
 
-    /// Create an account.
+    /// Create an account with the given or default description.
     ///
     /// Returns the editor for the created account.
     pub fn create(
         &mut self,
         key: Key,
-        description: String,
-        assets: Vec<AssetCode>,
-        records: Vec<RecordInfo>,
+        description: Option<String>,
```

Comment:
`Option` parameters to `create` are never necessary, because we could remove this parameter and instead do `accounts.create(key).with_description("whatever").save()`